test/functionalTest/cases/3389_forced_update_option/3389_forced_update_option_batch_bool.test

```diff
@@ -417,6 +418,7 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36}; cbnotif=[12])
     "subscriptionId": "REGEX([0-9a-f]{24})"
 }
 =======================================
+#SORT_END
```

Comment:
There is a race condition in the notifications triggered by the notifications. Some times the first one to reach accumulator-server.py is Slot1 and some times is Slot2. With this SORT_START / SORT_END clauses we make that, from a checking purposes, the order doesn't matter.

Same for other case in this file and in other .test files in test/functionalTest/cases/3389_forced_update_option

NTC (just informative)